changelog/README.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 24th January 2024
+* Fixed description of [Get all reservations (ver 2023-06-06)](../operations/reservations.md#get-all-reservations-ver-2023-06-06):
+  * Renamed `PartnerCompanyId` from `CompanyId` in the property list.
+  * Added `QrDataCode` and `CancellationReason` to the response.
+  * Removed `EnterpriseId` from the response (it was never part of the response).

```

Comment:
question: I will let @MikeAdamsMews to decide this one, but do you think the change like this should make it to the changelog? In this case we're fixing something which has been in the API already, but putting it into changelog makes it seem like we're changing the API.

Reply:
Yep, waiting on @MikeAdamsMews. I would not normally put it there but this information come from one of our partners 🤔

Reply:
I think the changes are significant enough that they should be recorded, but we should try and write the changelog entry to be clear that there are no changes to functionality. For example, add a line:
* Documentation-only, no change to API functionality

Reply:
@MikeAdamsMews What do you think of it now? I've wanted to put in as first thing to read as possible, to make it super clear.